radicle/src/cob/patch.rs

```diff
@@ -403,21 +409,25 @@ impl Revision {
     }
 }
 
+/// Patch state.
 #[derive(Debug, Clone, Copy, Default, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase", tag = "status")]
 pub enum State {
-    #[default]
-    Proposed,
-    Draft,
+    /// The patch is archived.
     Archived,
+    /// The patch is a draft.
+    Draft,
+    /// The patch is open.
```

Comment:
Comments are good, but how do these help? Could they instead describe what 'open' and 'draft' mean for the patch?

Reply:
I think the idea of a enum field comment is a short concise comment, I can see to make it clearer